tests/mocks.h

```diff
@@ -93,6 +93,11 @@ class MockBPFtrace : public BPFtrace
     return feature_->has_kprobe_multi();
   }
 
+  bool has_uprobe_multi(void)
+  {
+    return feature_->has_uprobe_multi();
```

Comment:
this reads from the running system? Is it possible to instead explicitly test both codepaths by manually setting has_uprobe_multi?

Reply:
Yeah, I just used the same approach as kprobe_multi does but you're right that unit tests should be system-independent. Let me fix that for both uprobe_multi and kprobe_multi.

Reply:
Fixed, please re-review, thanks! (I also updated the PR description)